src/client-agent/receiver.c

```diff
@@ -300,6 +301,11 @@ int receive_msg()
                                         }
                                     }
                                 }
+                                int j = 0;
+                                while (*(IGNORE_LIST + j)) {
+                                    free(*(IGNORE_LIST + j++));
+                                }
```

Comment:
To improve these lines of code, let's use the `w_FreeArray` macro with the `IGNORE_LIST` array:
- `w_FreeArray(IGNORE_LIST);`